onnxruntime/core/flatbuffers/ort.fbs

```diff
@@ -217,7 +217,9 @@ table SessionState {
 }
 
 table InferenceSession {
-  // This is the version of ORT which generates the model
+  // This is the ort format model version
```

Comment:
>ort [](start = 17, length = 3)

nit: ORT

Reply:
Will address this in my next other PR, need to get this in asap